### PR TITLE
{lib}[GCCcore/9.3.0] PyYAML v5.3, libyaml v0.2.2 w/ Python 2.7.17 + 3.8.2

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/569b46f1286ba2a4edb347bfa72ae90a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e7fd70fef971cdaeef5d29c64f91896e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bb2c1a09.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/3c992ae6dc97c030906de1e51894a767 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3abc73919abad1f05ce1785449eda638 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/3afa8af645d3b5a5a8b1f95fe3d3ed3e for a full test report.

File: easybuild/easyconfigs/p/PyYAML/PyYAML-5.3-GCCcore-9.3.0-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+options = {'modulename': 'yaml'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PyYAML/PyYAML-5.3-GCCcore-9.3.0-Python-3.8.2.eb]: This is the default for a PythonPackage please remove the whole sanity_check_paths section

Comment (branfosj) [reply easybuild/easyconfigs/p/PyYAML/PyYAML-5.3-GCCcore-9.3.0-Python-3.8.2.eb]: done

**Comment by branfosj** [comment]
Note that this has not been done as `multi_deps` as there is no Python 2.7.x at the moment. PyYAML 5.3 does still support Python 2.7.

**Comment by lexming** [comment]
@branfosj before merging this PR we should wait on the version of Python 2 that we target for `2020a`. If we do not wait for Python 2.7.18 (which seems likely), we will have an easyconfig for Python 2.7.17 in the next days and PyYAML could use `multideps`.

**Comment by boegel** [comment]
+1, let's promote this to `multi_deps` once #10145 is merged

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/664edb36736ee43637e82d7de37141b7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0306u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/b9bf9e1ab531db3c551404ef3e05e69a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9527de3c17427798dd5f5718fda36474 for a full test report.

**Comment by branfosj** [comment]
I've updated this to use `multi_deps` - new test reports are incoming.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bb2c1a09.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/1b630afd54e01ab373ac5b290c6f9b6e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/cae3402f98875fba23c6709d4b4d2897 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/42201b77e8be029a8411b80c34120e69 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/2efd92e93d228b9934b71ea1092cb00f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/36ceff01ab7b1264e4d4b3e6be55f3c7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/624de5c1a7ac7f0b904374f516b491d7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ubuntu-apps-builder2-its-rescomp-apps - Linux ubuntu 16.04, Intel Core Processor (Broadwell, no TSX), Python 2.7.12
See https://gist.github.com/0ecf84947df754d6ca6a13ba15619950 for a full test report.

**Pull request merged**